src/instructlab/config/init.py

```diff
@@ -111,14 +110,14 @@ def init(
         )
     click.echo(f"Generating `{helpers.DEFAULT_CONFIG}` in the current directory...")
     cfg = helpers.get_default_config()
-    model = splitext(basename(model_path))[0]
-    cfg.chat.model = model
-    cfg.generate.model = model
+    cfg.chat.model = model_path
+    cfg.generate.model = model_path
```

Comment:
It looks like this belonged in an earlier commit?

Reply:
removed